src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -309,9 +313,13 @@ internal sealed class MeterProviderSdk : MeterProvider
                                     listener.EnableMeasurementEvents(instrument, metrics);
                                 }
                             }
-                        }
 
-                        OpenTelemetrySdkEventSource.Log.MeterProviderSdkEvent($"Completed publishing Instrument = \"{instrument.Name}\" of Meter = \"{instrument.Meter.Name}\".");
+                            OpenTelemetrySdkEventSource.Log.MeterProviderSdkEvent($"Enabled measurements for Instrument = \"{instrument.Name}\" of Meter = \"{instrument.Meter.Name}\".");
```

Comment:
Should not log if [this](https://github.com/open-telemetry/opentelemetry-dotnet/blob/ef0c9217087bfca0d6fe9206ccde343eb4e5738c/src/OpenTelemetry/Metrics/MeterProviderSdk.cs#L298) return `null`

Reply:
Good catch. Fixed this.